src/lib/fb.js

```diff
@@ -195,6 +203,7 @@ export async function saveUserUrl(url, auditedOn = null) {
 export async function signIn() {
   let user = null;
   try {
+    initialize();
```

Comment:
Can you add a short comment on why the lazy loading is important here, mentioning indexdb?

Reply:
Added (also added a comment above to the `initialize()` function.